openlibrary/plugins/importapi/code.py

```diff
@@ -62,9 +62,13 @@ def parse_data(data):
     :param str data: Raw data
```

Comment:
`parse_data` is only called from one place https://github.com/internetarchive/openlibrary/blob/10a05f78ee40a5137163c54f03c833203db064e0/openlibrary/plugins/importapi/code.py#L121-L124

is it true that `web.data()` can return either a `str` or `bytes`? I'm struggling to find up to date info on web.py expectations under Python 3

Reply:
The new proposed approach is assume that `data` can only be bytes so it must be compared with bytes.